cmd/scollector/collectors/collectors.go

```diff
@@ -62,6 +62,21 @@ const (
 )
 
 const (
+	bgpPeerState               = "bgp.peer.state"
```

Comment:
you can actually define these inside of the collector function that uses them. I think doing that may help keep the collectors package from having quite so many symbols defined at package scope.